.rhcicd/build_deploy.sh

```diff
@@ -15,7 +15,26 @@ if [[ -z "$RH_REGISTRY_USER" || -z "$RH_REGISTRY_TOKEN" ]]; then
     exit 1
 fi
 
-DOCKER_CONF="$PWD/.docker"
+# Set the job's build directory to something else other than the workspace, in
+# order to avoid leaks.
+readonly job_directory_path=$(mktemp --directory -p "${HOME}" -t "jenkins-${JOB_NAME}-${BUILD_NUMBER}.XXXXXX")
```

Comment:
Why do we need `.XXXXXX` at the end of the directory name?

Reply:
The `.XXXXX`s get replaced with random characters, to avoid collisions. With the job name and build number I feel like it is very unlikely that we will suffer from any collision, but since the `mktemp` utility gives us that nicety, I felt like we should use it.